metabase/metabase
# Search Milestone 3: Native Query Search Toggle

`search_native_query` filter for Search Milestone 3: #27982 

### Description

Adds `search_native_query` filter to the search sidebar. The filter reuses the logic the `verified` filter for the toggle component, and searches within SQL/native query text using the user's text query.


### How to verify

Create two questions - this will help eliminate extra results that won't be useful in verifying this PR:
* A native question with really specific text, like `SELECT 'SomethingIsTrulySpecific'` and give it a generic name like `Query`.
* Another question with a substring of that specific text as the name (like `SomethingIsTrulySpecific`)

In the search bar, type that text. You should only see one result: the question with the specific name _(not the question with the generic name)_.

Now, click on the `Search the contents of SQL queries`.
_Note: this will change to `native queries` if you have a Mongo or NoSQL database in your system as well._

You should see two results: the question with the really specific name, and the question with the really generic name. This means that, in the example, searching for `SomethingIsTrulySpecific` found a question with the name of `SomethingIsTrulySpecific`, as well as a question whose query contained `SomethingIsTrulySpecific`.


### Tests
